diff --git a/src/EventSource.js b/src/EventSource.js
index 6a7b146ef5872f72eda3eb52c265abb996fd7014..e43fcf8a32a310166f1a7d2e18b2050a92c7a49f 100644
--- a/src/EventSource.js
+++ b/src/EventSource.js
@@ -214,7 +214,7 @@ class EventSource {
           this.interval = retry;
         }
       } else if (line.startsWith('data')) {
-        data.push(line.replace(/data:?\s*/, ''));
+        data.push(line.replace(/^data:/, ''));
       } else if (line.startsWith('id')) {
         id = line.replace(/id:?\s*/, '');
         if (id !== '') {
